libs/npm/core/src/utils/getDistinct.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Generates a new array of distinct items based on the specified 'predicate'.
+ * @param values An array of values.
+ * @param predicate Predicate to extract value from item in array.
+ * @returns A new array with distinct values based on the predicate.
+ */
+export const getDistinct = (values: any[], predicate: (item: any) => any) => {
```

Comment:
@jdtoombs We have a simple way to get distinct items from an array now.

Reply:
I like!